### PR TITLE
Update netci.groovy to use Dev15 RC

### PR DESCRIPTION
**Pull request by tmeschter**
Update netci.groovy to use Dev15 RC for all builds.

### CODE REVIEW

**Comment by tmeschter** [comment]
@tannergooding @jasonmalinowski @jaredpar @Pilchie Please review.

File: netci.groovy [hunk 1]
```diff
@@ -197,7 +197,7 @@ set TMP=%TEMP%
 
   def triggerPhraseOnly = true
   def triggerPhraseExtra = "open-vsi"
-  Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15-rc')
+  Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15-rc-rc')
```
Comment (jaredpar) [inline netci.groovy]: This seems wrong

Comment (tmeschter) [inline netci.groovy]: Indeed. Let's fix that...

Comment (tmeschter) [inline netci.groovy]: ...and done.

File: netci.groovy [hunk 2]
```diff
@@ -178,7 +178,7 @@ commitPullList.each { isPr ->
 
   def triggerPhraseOnly = false
   def triggerPhraseExtra = "microbuild"
-  Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15')
+  Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15-rc')
```
Comment (jaredpar) [inline netci.groovy]: I thought these machines were Dev14 + Dev15RC.  If it's Dev15 only then yeah can't do this for a few of these jobs.

**Comment by tmeschter** [comment]
I'm going to back out the changes to `microbuild` and `perf-correctness` and merge the rest.

**Comment by jaredpar** [comment]
please do NOT merge this .

**Comment by jaredpar** [comment]
If this really has Dev15 only then we can't merge this until we are confident the unit tests will pass on dev15 only.  So far the reports I've seen are completely the opposite: tests fail in Dev15 only environments.

**Comment by tmeschter** [comment]
@jaredpar Good to know. I'll hold off.

**Comment by DustinCampbell** [comment]
@tmeschter: what's the plan for this PR?

**Pull request closed without merging**